app/src/main/java/com/zulip/android/activities/RecyclerMessageAdapter.java

```diff
@@ -427,6 +439,64 @@ public void onClick(View view) {
                 setUpGravatar(message, messageHolder);
                 setUpTime(message, messageHolder);
                 setUpStar(message, messageHolder);
+                try {
+                    messageHolder.reactionsTable.removeAllViews();
+                    if (message.getReactions().isEmpty()) {
+                        return;
+                    }
+
+                    // Calculate number of reactions in each row
+                    int messageWidth = messageHolder.messageTile.getContext().getResources().getDisplayMetrics().widthPixels;
+                    int reactionWidth = convertDpToPixel(64);
+                    int numOfReactions = messageWidth / reactionWidth;
+                    TableRow row = new TableRow(messageHolder.messageTile.getContext());
```

Comment:
Not sure if I got your point here. Do you mean this [context](https://github.com/Sam1301/zulip-android/blob/ede40c540d898ab39e999c7ddf15f2e232892270/app/src/main/java/com/zulip/android/activities/RecyclerMessageAdapter.java#L114) ?
I intended to pass the messagetile context since new Table rows need to be create within a single messagetile.
According to the docs,
```
/**
     * <p>Creates a new TableRow for the given context and with the
     * specified set attributes.</p>
     *
     * @param context the application environment
     * @param attrs a collection of attributes
     */
    public TableRow(Context context, AttributeSet attrs) {
        super(context, attrs);
        initTableRow();
    }
```